services/src/main/java/org/keycloak/broker/saml/SAMLIdentityProvider.java

```diff
@@ -343,10 +343,16 @@ private JaxrsSAML2BindingBuilder buildLogoutBinding(KeycloakSession session, Use
     public Response export(UriInfo uriInfo, RealmModel realm, String format) {
         try
         {
-            URI authnBinding = JBossSAMLURIConstants.SAML_HTTP_REDIRECT_BINDING.getUri();
+            URI authnResponseBinding = JBossSAMLURIConstants.SAML_HTTP_REDIRECT_BINDING.getUri();
 
-            if (getConfig().isPostBindingAuthnRequest()) {
-                authnBinding = JBossSAMLURIConstants.SAML_HTTP_POST_BINDING.getUri();
+            if (getConfig().isPostBindingResponse()) {
```

Comment:
I think we should maintain using `isPostBindingAuthnRequest` here. In theory both values should be the same but the one that really is forcing to use the post binding is the `isPostBindingAuthnRequest` (see [here](https://github.com/keycloak/keycloak/blob/21.0.1/services/src/main/java/org/keycloak/broker/saml/SAMLIdentityProvider.java#L170) and then [here](https://github.com/keycloak/keycloak/blob/21.0.1/services/src/main/java/org/keycloak/broker/saml/SAMLIdentityProvider.java#L196-L200)). The ` isPostBindingResponse` is only used to set the attribute in the request ([here](https://github.com/keycloak/keycloak/blob/21.0.1/services/src/main/java/org/keycloak/broker/saml/SAMLIdentityProvider.java#L135-L137)).

Modify the tests accordingly please.